packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.service.js

```diff
@@ -2,8 +2,9 @@ import { SOFTPHONE_TYPE } from './softphone.constants';
 
 export default class SofpthoneService {
   /* @ngInject */
-  constructor($http) {
+  constructor($http, $window) {
     this.$http = $http;
+    this.$window = $window;
```

Comment:
```suggestion
  constructor($http, $window, $q) {
    this.$http = $http;
    this.$window = $window;
    this.$q = $q;
```

Reply:
Same as below, why is it better to use $q here?